### PR TITLE
Completion of the DBSync job.

### PR DESCRIPTION
**Pull request by dkehn**

### CODE REVIEW

File: templates/designateapi/bin/bootstrap.sh [hunk 1]
```diff
@@ -15,6 +15,7 @@
 # under the License.
 set -ex
 
-OPTS="--config-file /etc/designate/designate.conf"
-designate-manage database ${OPTS} upgrade head
+# Note(dkehn) - this maybe need to change back to upgrade head?
+# designate-manage database upgrade head
+designate-manage database sync
```
Comment (johnsom) [inline templates/designateapi/bin/bootstrap.sh:20]: Curious why this needs to be a shell script. Couldn't this be run by the golang directly?

**Review by johnsom — COMMENTED** [review]
I hate PRs because I have other comments, but it won't let me add them as they were not changed lines. Anyway, here are some comments on what I can comment on.

File: templates/designateapi/bin/bootstrap.sh [hunk 2]
```diff
@@ -15,6 +15,7 @@
 # under the License.
 set -ex
 
-OPTS="--config-file /etc/designate/designate.conf"
-designate-manage database ${OPTS} upgrade head
+# Note(dkehn) - this maybe need to change back to upgrade head?
+# designate-manage database upgrade head
```
[the remark this answers is not in the document]
Comment (dkehn) [reply templates/designateapi/bin/bootstrap.sh:19]: Will change have confirm it works

File: pkg/designate/dbsync.go [hunk 3]
```diff
@@ -28,6 +28,8 @@ import (
 const (
 	// DBSyncCommand -
 	DBSyncCommand = "/usr/local/bin/kolla_set_configs && /usr/local/bin/kolla_start"
+	// NOTE(dkehn): the following is here for debugging purposes!!!!!!!!!!!!!!
+	// DBSyncCommand = "/bin/sleep 99999"
```
[the remark this answers is not in the document]
Comment (dkehn) [reply pkg/designate/dbsync.go:32]: Removing the debugging hint.

File: templates/designateapi/config/designate.conf [hunk 4]
```diff
@@ -76,8 +80,10 @@ auth_type=password
 # memcache_use_advanced_pool=True
 # memcached_servers=FIXMEhost1:11211
 # region_name=regionOne
-interface=internal
+
+# interface=internal
+
+cafile=/opt/stack/data/ca-bundle.pem
```
[the remark this answers is not in the document]
Comment (dkehn) [reply templates/designateapi/config/designate.conf:86]: Will remove

File: templates/designateapi/config/designate.conf [hunk 5]
```diff
@@ -5,59 +5,63 @@ debug=True
...
-topics = notifications
-driver = messagingv2
+topics=notifications
+driver=messagingv2
 
 [oslo_concurrency]
-lock_path = /opt/stack/data/designate
+lock_path=/opt/stack/data/designate
 
 [health_manager]
```
[the remark this answers is not in the document]
Comment (dkehn) [reply templates/designateapi/config/designate.conf:66]: this is our (designate) config file.

**Pull request merged**